CommonClient.py

```diff
@@ -278,6 +279,17 @@ def __init__(self, server_address: typing.Optional[str], password: typing.Option
         # execution
         self.keep_alive_task = asyncio.create_task(keep_alive(self), name="Bouncy")
 
+        if gui_enabled:
+            from kvui import GameManager
+
+            class TextManager(GameManager):
+                logging_pairs = [
+                    ("Client", "Archipelago")
+                ]
+                base_title = "Archipelago Text Client"
```

Comment:
i know the logging_pairs assignment was already there before your PR, but it's exactly the same as the base class, so doesn't really make much sense to redeclare
```suggestion
            class TextManager(GameManager):
                base_title = "Archipelago Text Client"
```